docs/prod-fluo-setup.md

```diff
@@ -67,6 +67,11 @@ NOTE - All properties that have a default are set with it.  Uncomment a property
 to use a value different than the default.  Properties that are unset and uncommented must be
 set by the user.
 
+To run, Fluo requires the Hadoop, Accumulo, and Zookeeper jars for the versions
```

Comment:
The `Obtain a distribution` section of this file should be updated to reflect that Fluo distributions are no longer built for specific versions of Hadoop or Accumulo.  Below is some sample text.  Feel free to modify:

```
Before you can install Fluo, you will need to obtain a distribution tarball.  It is
recommended that you download the [latest release][release].  You can also build
a distribution from the master branch by following these steps which create a tarball
in `modules/distribution/target`:

    git clone https://github.com/apache/incubator-fluo.git
    cd fluo/
    mvn package
```